common/src/main/java/dev/ftb/mods/ftblibrary/util/MapUtils.java

```diff
@@ -0,0 +1,31 @@
+package dev.ftb.mods.ftblibrary.util;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MapUtils {
```

Comment:
How does this work when the sorted data is just put into a new `HashMap` (which doesn't have any ordering semantics) ?

Wouldn't this need a `LinkedHashMap` or similar?

Reply:
The code did what I wanted it to do, converted to a LinkedHashMap to be safe